kartograf/context.py

```diff
@@ -48,7 +48,10 @@ def __init__(self, args):
             self.epoch_dir = str(int(utc_time_now))
             self.epoch_datetime = datetime.utcfromtimestamp(int(utc_time_now))
 
-        cwd = os.getcwd()
+        if hasattr(self.args, 'cwd'):
```

Comment:
Hm, I don't see what this is needed for.

Reply:
yea this was a temporary hack around the fact that using `tmp_path` doesn't actually move the codepath, just runs the test in that tmp_path. so `os.getcwd()` was generating test data in the code path. Fixed it below in [b2a23ff](https://github.com/asmap/kartograf/pull/49/commits/b2a23ff3251a641d2f3d0d773e907e61fd32c36d).